index.html

```diff
@@ -60,12 +60,21 @@ <h2>Welcome to ROS Index</h2>
   </div>
  <hr>
   <div class="row">
-  	<h2>Release announcements</h2>
+  	<h2><a href="http://docs.ros.org/en/rolling/Releases.html">Active Releases</a></h2>
+    <p>
+      <a href="https://wiki.ros.org/melodic">ROS Melodic</a>
+    </p>
     <p>
       <a href="https://wiki.ros.org/noetic">ROS Noetic</a>
     </p>
     <p>
-      <a href="https://index.ros.org/doc/ros2/Releases/Release-Foxy-Fitzroy/">ROS 2 Foxy</a>
+      <a href="http://docs.ros.org/en/rolling/Releases/Release-Dashing-Diademata.html">ROS 2 Dashing</a>
```

Comment:
I think it probably makes more sense for this to point to the 'dashing' branch, like:

```suggestion
      <a href="https://docs.ros.org/en/dashing/Releases/Release-Dashing-Diademata.html">ROS 2 Dashing</a>
```